packages/app/src/app/page.tsx

```diff
@@ -1,10 +1,19 @@
+'use client'
+
+import React from 'react'
 import '@marginly/ui/styles/reset.css'
+import { LayoutSwitcher } from '../../components/mobile-placeholder.tsx/styled'
+import MobilePlaceholder from './mobile-placeholder'
 
 export default function Home() {
   return (
     <main>
-      <h2>Slender App</h2>
-      <p>TBD</p>
+      <LayoutSwitcher>
```

Comment:
```suggestion
      <LayoutSwitcher mobile={<MobilePlaceholder />} desktop={<div>app</div>} />
```
Лучше логику по отображению мобилы или десктопа спрятать в LayoutSwitcher компонент

Reply:
Fixed